app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,7 @@
 <% provide :body do %>
-<%= render 'shared/header' %>
+<div class="lux">
```

Comment:
@sdellis we added a second lux load... is this bad? There was one already for the order manager

Reply:
They shouldn't run into each other, but yeah @sdellis would know better.

Reply:
👍 confirmed via slack: "yes you can load more than one lux mounting element. I think there may be an issue with multiple mounting elements  in the static build"